UI_UX/Frontend/src/components/Achievements.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import '../styles/Achievements.css';
+
+export class Achievements extends Component {
+  render() {
+    return (
+      <ol className="Achievements">
+        {this.props.achievements.map(milestone => (
+          <li key={milestone.endpoint} className="Achievements__list-item">
```

Comment:
Probably no need to nest all these elements, and remember `--` is only for modifiers. `title` and `description` are elements.

So we could just do:
```
Achievements__list-item-wrapper
Achievements__list-item-title
Achievements__list-item-description
```